users/tests.py

```diff
@@ -25,6 +25,14 @@ def test_create_team_without_title(self):
         with pytest.raises(Exception):
             Team.objects.create(description="Best team ever")
 
+    def test_create_teams_with_same_name(self):
+        assert len(Team.objects.all()) == 0
+        Team.objects.create(name="Team1", description="This team will be created")
+        assert len(Team.objects.all()) == 1
+        with pytest.raises(Exception):
+            Team.objects.create(name="Team1", description="This team will be created")
+            assert len(Team.objects.all()) == 1
```

Comment:
I think you can add here some message : assert len(Team.objects.all()) == 1,  f"this team {Team.object.name} created twice"